lib/studies/reply/adaptive_nav.dart

```diff
@@ -665,6 +675,88 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
   }
 }
 
+class _BottomAppBarActionItems extends StatelessWidget {
+  const _BottomAppBarActionItems({@required this.drawerVisible})
+      : assert(drawerVisible != null);
+
+  final bool drawerVisible;
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<EmailStore>(
+      builder: (context, model, child) {
+        final onMailView = model.currentlySelectedEmailId >= 0;
+
+        return AnimatedSwitcher(
+          duration: const Duration(milliseconds: 350),
+          transitionBuilder: (child, animation) => ScaleTransition(
+            alignment: const Alignment(1, 0),
+            child: child,
+            scale: animation,
+          ),
+          child: drawerVisible
+              ? Row(
+                  key: UniqueKey(),
+                  mainAxisSize: MainAxisSize.max,
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: [
+                    IconButton(
```

Comment:
Is the Row necessary if it's only got one child?

Reply:
It was a hacky fix for some layout issues, updated with a solution that avoids this.